### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by johnsaigle** [dustMason/op1.fun.app#4]
What is the code license for this repository?

**Comment by terje** [comment]
This would be great to know, what a super companion app for the site. Would love to contribute.

**Comment by dustMason** [comment]
I need to add a LICENSE file! License should be considered as friendly and open as possible. I think MIT license is good for that?

**Comment by johnsaigle** [comment]
Yep, MIT provides pretty much full access for anyone to do anything.

GPLv3 is also a good choice because it makes sure that other people who use the software have to keep it open-source for other people to use too! (e.g. it might prevent a company from stealing your source code, making proprietary modifications, selling it as a product, and restricting from users from seeing the source code). MIT is more open though.
